### PR TITLE
dbg and util.secondsToClock unit tests

### PR DESCRIPTION
**Pull request by Frenzie**
* dbg_spec: setVerbose test 
* [chore] Rework util spec, rework util.secondsToClock: round seconds to minutes in 00:00 mode + spec

Most of the tests in util_spec were the wrong way around.
It's `assert(expected, given)`.

### CODE REVIEW

File: spec/unit/util_spec.lua [hunk 1]
```diff
@@ -298,15 +299,46 @@ describe("util module", function()
                     "Deux parties.\nPrologue.Désespérée, elle le tue...\nPremière partie. Sur la route & dans la nuit")
     end)
     it("should return true on empty dir", function()
-        assert.is_equal(util.isEmptyDir(DataStorage:getDataDir() .. "/data/dict"), -- should be empty during unit tests
-                    true)
+        assert.is_equal(true,
```
Comment (Frenzie) [inline spec/unit/util_spec.lua]: Should be is_true.

File: frontend/util.lua [hunk 2]
```diff
@@ -80,8 +80,9 @@ function util.secondsToClock(seconds, withoutSeconds)
             return "00:00:00";
         end
     else
+        local round = withoutSeconds and require("optmath").round or math.floor
```
Comment (Frenzie) [inline frontend/util.lua]: @robert00s Any thoughts on this change btw? It's used a bunch in Statistics (just for display afaik). I thought it was kind of odd to always floor.

00:01:59 → 00:02 (really?)
now:
00:01:29 → 00:01
00:01:30 → 00:02

Comment (robert00s) [reply frontend/util.lua]: This change look good for me :+1:

Comment (Frenzie) [reply frontend/util.lua]: Alright, thanks. :-)

**Pull request merged**